CMakeLists.txt

```diff
@@ -388,11 +388,9 @@ find_package(Boost 1.70.0 REQUIRED COMPONENTS filesystem log log_setup thread
                                               program_options system)
 
 # diskhash
-if(NOT CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
No problem, there is still an error for MacOSX, but seems to be a trivial thing to be fixed on DiskHash library. Let's see if there will be more errors on Windows, so I can send a new pack of changes.

Reply:
Sounds good, thanks for your swift intervention and yeah let's see how the Windows tests behave first.

Reply:
Yes, sure. BTW, do you think the current failure on develop branch will also impact this PR? Does this branch have the same clang change?

Reply:
Hmm no, not really, that one got solved anyway. Let's look into the latest diskhash-related build fails on Win and other than that we should be fine.

Reply:
DiskHash is indeed broken on Windows. It will require a more deep investigation. I suggest making this PR a draft, so we can resume it once we solve the DH problems.

Reply:
Agree, let's put it on hold therefore.